pkg/operation/botanist/component/vpa/admissioncontroller.go

```diff
@@ -208,6 +208,7 @@ func (v *vpa) reconcileAdmissionControllerDeployment(deployment *appsv1.Deployme
 						fmt.Sprintf("--port=%d", admissionControllerPort),
 						"--register-webhook=false",
 					},
+					LivenessProbe: getDefaultLivenessProbe(),
```

Comment:
After looking more closely at the admission controller, I see that its liveness probe does two things:
1) Spend some time to determine if the period since last activity exceed the specified timeout period (one minute)
2) Throw that information away, and return HTTP 200 "OK" unconditionally.

I don't see how the liveness probe would have any effect, except when the probe itself times out. That could happen, but, networking issues aside, I expect it to be quite rare (e.g. extreme CPU starvation/exclusion due to some high priority component spinning in multiple tight loops, or kernel/driver issue resulting in a tight loop/deadlock; also, a hypothetical extremely slow memory allocation, due to GC or system heap compaction - don't know if that's a thing on Linux). Basically the thread scheduler on the host being prevented from effectively scheduling threads. I've seen many such issues on my desktop machines over the years, but I think they should be rare on server class machines. I'd say that at this point in time, we're implementing admission controller probing as much to benefit from potential future enhancements to the probe, as from its current function.